package.json

```diff
@@ -31,22 +28,13 @@
     "captains-log": "0.11.11",
     "colors": "0.6.2",
     "commander": "2.1.0",
-    "compression": "1.6.1",
-    "cookie-parser": "1.4.1",
     "debug": "2.2.0",
-    "ejs": "0.8.8",
-    "ejs-locals": "1.0.2",
-    "express": "4.13.4",
-    "express-handlebars": "1.0.3",
     "fs-extra": "0.8.1",
     "glob": "3.2.11",
     "include-all": "0.1.6",
```

Comment:
Ditto.
It's not in `waterline`'s package.json, but it _is_ in waterline's `npm-shrinkwrap`... (This might be another reason not to have shrinkwrap files in modules, btw – it can break npm's auto-deduping.)

It appears to be used by `sails/node_modules/waterline/node_modules/waterline-criteria`, so it looks like it should be installed at that level.